README.md

```diff
@@ -3,3 +3,91 @@
+  🤖 [Auth Forms](https://www.hyperui.dev/components/application-ui/login-forms)  
+  🤖 [Pagination](https://www.hyperui.dev/components/application-ui/pagination)  
+  🤖 [Progress](https://www.hyperui.dev/components/application-ui/progress)  
+  🤖 [Radio Groups](https://www.hyperui.dev/components/application-ui/radio-groups)  
+  🤖 [Selects](https://www.hyperui.dev/components/application-ui/selects)  
+  🤖 [Side Menu](https://www.hyperui.dev/components/application-ui/side-menu)  
+  🤖 [Stats](https://www.hyperui.dev/components/application-ui/stats)  
+  🤖 [Steps](https://www.hyperui.dev/components/application-ui/steps)  
+  🤖 [Tables](https://www.hyperui.dev/components/application-ui/tables)  
+  🤖 [Tabs](https://www.hyperui.dev/components/application-ui/tabs)  
+  🤖 [Textareas](https://www.hyperui.dev/components/application-ui/textareas)  
+  🤖 [Toggles](https://www.hyperui.dev/components/application-ui/toggles)  
+  🤖 [Vertical Menu](https://www.hyperui.dev/components/application-ui/vertical-menu)  
+  
+### Marketing
+📣 [Announcements](https://www.hyperui.dev/components/marketing/announcements)  
+📣 [Banners](https://www.hyperui.dev/components/marketing/banners)  
+📣 [Blog Cards](https://www.hyperui.dev/components/marketing/blog-cards)  
+📣 [Buttons](https://www.hyperui.dev/components/marketing/buttons)  
+📣 [Cards](https://www.hyperui.dev/components/marketing/cards)  
+📣 [CTAs](https://www.hyperui.dev/components/marketing/ctas)  
+📣 [FAQs](https://www.hyperui.dev/components/marketing/faqs)  
+📣 [Footers](https://www.hyperui.dev/components/marketing/footers)  
+📣 [Forms](https://www.hyperui.dev/components/marketing/forms)  
+📣 [Header](https://www.hyperui.dev/components/marketing/headers)  
+📣 [Popups](https://www.hyperui.dev/components/marketing/popups)  
+📣 [Pricing](https://www.hyperui.dev/components/marketing/pricings)  
+📣 [Sections](https://www.hyperui.dev/components/marketing/sections)  
+📣 [Stats](https://www.hyperui.dev/components/marketing/stats)  
+📣 [Testimonials](https://www.hyperui.dev/components/marketing/testimonials)  
+
+### eCommerce
+🛍️ [Carts](https://www.hyperui.dev/components/ecommerce/carts)  
+🛍️ [Collection Cards](https://www.hyperui.dev/components/ecommerce/collection-cards)  
+🛍️ [Collection Filters](https://www.hyperui.dev/components/ecommerce/collection-filters)  
+🛍️ [Featured Filters](https://www.hyperui.dev/components/ecommerce/featured-sections)  
+🛍️ [Product Cards](https://www.hyperui.dev/components/ecommerce/product-cards)  
+🛍️ [Product Collections](https://www.hyperui.dev/components/ecommerce/product-collections)  
+🛍️ [Quantity Inputs](https://www.hyperui.dev/components/ecommerce/quantity-inputs)  
+
+## Contributing
+HyperUI accepts contributions. For guidance on how to contribute to the project, a detailed [contributing guide](https://www.hyperui.dev/blog/how-to-contribute) is available on the HyperUI [blog](https://www.hyperui.dev/blog).
```

Comment:
Once again, there's some in the FAQs about this (Sorry, I should have linked to that):

> You can contribute by adding new components, fixing bugs or improving the documentation. Please create an issue before starting any work.

I'm missing some information about the blog post which outlines how to contribute in that text, it would be good to get that added in somehow, maybe a little rewording?

Reply:
Be good to get that link added to the FAQs as well, possibly another PR if you want it for Hacktoberfest?

Reply:
Another PR for Hacktoberfest sounds great @markmead. I will make the other requested changes and update this PR 🙂